internal/controller/postgrescluster/postgres.go

```diff
@@ -71,12 +71,19 @@ func (r *Reconciler) generatePostgresUserSecret(
 		intent.Data["password"] = existing.Data["password"]
 		intent.Data["verifier"] = existing.Data["verifier"]
 	}
-	if len(intent.Data["password"]) == 0 || len(intent.Data["verifier"]) == 0 {
-		password, err := util.GeneratePassword(util.DefaultGeneratedPasswordLength)
+
+	var password string
+	// When password is unset, generate a new one
```

Comment:
⛏️ L69 says "Generate [password and verifier] when either is missing" but that's not true anymore.